src/ert/run_models/multiple_data_assimilation.py

```diff
@@ -191,7 +191,7 @@ def run_experiment(
 
         self.setPhaseName("Post processing...")
 
-        self.setPhase(self.number_of_iterations, "Experiment completed.")
+        self.setPhase(self.phaseCount(), "Experiment completed.")
```

Comment:
Perhaps just write `self.start_iteration + self.number_of_iterations` instead of `self.phaseCount()`, a bit less indirect. In a separate PR it would then maybe make sense to rename `number_of_iterations` to `number_of_iterations_from_start` to make it more explicit.